packages/local_auth/local_auth/lib/error_codes.dart

```diff
@@ -24,3 +24,6 @@ const String lockedOut = 'LockedOut';
 /// Indicates the API is locked out more persistently than [lockedOut].
 /// Strong authentication like PIN/Pattern/Password is required to unlock.
 const String permanentlyLockedOut = 'PermanentlyLockedOut';
+
+/// Indicates that the biometricOnly parameter can't be true on Windows
+const String biometricOnlyNotSupported = 'biometricOnlyNotSupported';
```

Comment:
Oops, we made the string a different case than the other errors. Oh well, hopefully people will use the constant.

Reply:
Ops. I don't think it is worth fixing such a small thing now, right?

Reply:
Nope, not with the constant as the recommended approach anyway.